project/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
is this needed?

Reply:
Depends.  I was using the sameple integration to verify that `HTTP -> HTTPS` redirects were working correctly.  It's needed for that (otherwise the webview won't got to any plain HTTP site), otherwise not.

Thought I'd leave it just so it's easier to repeat that sort of test in the future?

Reply:
okies; should be fine1